lua/plugins/init.lua

```diff
@@ -1,6 +1,9 @@
 return {
+  { "jay-babu/mason-nvim-dap.nvim", enabled = false },
+  { "nvim-neo-tree/neo-tree.nvim", enabled = false },
+
   { "akinsho/toggleterm.nvim", opts = { terminal_mappings = false } },
-  { "rcarriga/nvim-notify",    opts = { background_colour = "#000000" } },
+  { "rcarriga/nvim-notify", opts = { background_colour = "#000000" } },
   {
     "ethanholz/nvim-lastplace",
     event = "BufRead",
```

Comment:
Are you fine with me removing this? We have our own lastplace implementation built in AstroNvim which also does fold states and also detects commit/rebase buffers. This shouldn't be necessary.

Reply:
Removed!